tests/integration/targets/lookup_passwordstore/tasks/tests.yml

```diff
@@ -60,3 +60,29 @@
   assert:
     that:
       - readpass == newpass
+
+# As inserting multiline passwords on the commandline would require something
+# like expect, simply create it by using default gpg on a file with the correct
+# structure.
+- name: Create the YAML password content
+  set_fact:
+    yamlpass: |
+      testpassword
+      key: |
+        multi
+        line
+
+- name: Create a password-file that contains YAML
+  shell: echo "{{ yamlpass }}" > ~/.password-store/test-yaml-pass
```

Comment:
It's easier (and safer) to use the `copy` module with its `content:` option for this.